src/modules/notes/NotesOverview.js

```diff
@@ -17,19 +17,18 @@ const SpotNotesOverview = props => {
   return (
     <View>
       {savedNote ? <View style={noteStyles.notesOverviewContainer}>
```

Comment:
put parentheses around the View so the opening tag <View lines up with the closing tag </View>